### ISSUE TITLE
Generating in sections

### ISSUE THREAD

**Issue by bowmatty** [HashLips/hashlips_art_engine#1364]
Hello, 

Is it possible for me to generate in sections, i want to create 1000 and then run again and create another 1000 and start from the continuing number. Example, generate 1-1000, and then a separate 1001-2000 

Thank you  :)

**Comment by bolshoytoster** [comment]
@bowmatty you can use [my fork](https://github.com/bolshoytoster/hashlips_art_engine) and just change `growEditionSizeTo` to whatever you want the full collection to be, i.e. add 1000 to it each time.

**Comment by bowmatty** [comment]
And will that just continue on..i.e 1000 then next gen will be 1001-2000 etc?

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by bolshoytoster** [comment]
@bowmatty yes.

**Comment by bowmatty** [comment]
Thanks mate,
How do i use your fork? sorry noob here

**Comment by bolshoytoster** [comment]
@bowmatty the same way you use the main hashlips, you can get a zip [here](https://github.com/bolshoytoster/hashlips_art_engine/archive/refs/heads/main.zip).

**Comment by bowmatty** [comment]
Thanks mate

If i add 1000 to it and then generate, will the next gen continue from 1001-2000? My current fork just deletes and recreate from 1 each time, not continuous if you know what i mean.

**Comment by bolshoytoster** [comment]
@bowmatty yes.

**Comment by bowmatty** [comment]
Do you have a screenshot of how the "growEditionSizeTo" should look when generating each 1000? so i can get an idea. 

Thanks again

**Comment by bolshoytoster** [comment]
@bolshoytoster you just set it to whatever you want the collection size to be each time, i.e. 1000, then 2000, then 3000

**Comment by bowmatty** [comment]
Ok great, ill give that ago. 
Thanks again

**Comment by bowmatty** [comment]
I got multiple errors when running this fork.. 

PS C:\Users\Matt\Downloads\hashlips_art_engine-main> npm run generate 
npm WARN config global `--global`, `--local` are deprecated. Use `--location=global` instead.
npm ERR! code ENOENT
npm ERR! syscall open
npm ERR! path C:\Users\Matt\Downloads\hashlips_art_engine-main/package.json
npm ERR! errno -4058
npm ERR! enoent ENOENT: no such file or directory, open 'C:\Users\Matt\Downloads\hashlips_art_engine-main\package.json'
npm ERR! enoent This is related to npm not being able to find a file.
npm ERR! enoent

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\Matt\AppData\Local\npm-cache\_logs\2022-08-18T07_51_15_085Z-debug-0.log

**Comment by bolshoytoster** [comment]
@bowmatty you're in the wrong folder. Run `Get-ChildItem -Directory` to get the name, then run `cd ` + the name to change into it.

**Comment by bowmatty** [comment]
Yep i ran the Get-ChildItem -Directory, which name do i need?

**Comment by bolshoytoster** [comment]
@bowmatty the one you get from running `Get-ChildItem -Directory`.

**Comment by LaughingElephant** [comment]
Love this fork. Any suggestions to be able to update the base URI without updating the name? Intent of use is to have different names within the same collection - but need to be able to update the base URI without it also changing the names.

**Comment by bolshoytoster** [comment]
@LaughingElephant just add a `//` to the start of line ~43 in `utils/update_info.js`:
https://github.com/bolshoytoster/hashlips_art_engine/blob/708bf21426b11f3592697bde959caadcf11a722a/utils/update_info.js#L43

**Comment by LaughingElephant** [comment]
@bolshoytoster Easy enough lol, that worked thanks man. 

If you figure out how to set each 'growEditionSize' container to its own name in the metadata hit me up. Would be nice to shuffle the different names around.

**Comment by elpigeontest** [comment]
Hi @bolshoytoster your fork could work for this scenario?

I'm making a collection that use two different base drawings. The traits are the "same", but I can't use the same drawing of the traits because the bases have different sizes.
Is there any way for me to generate the collection with some conditionals? Like, If base layer is X, then use these traits, or if base layer is Y then use these other traits?

**Comment by bolshoytoster** [comment]
@elpigeontest you could have two different `layersConfigurations`, one with traits x and one with y. My fork doesn't support having different image sizes though.

**Comment by elpigeontest** [comment]
> @elpigeontest you could have two different `layersConfigurations`, one with traits x and one with y. My fork doesn't support having different image sizes though.

Thank you! The problem is in this way the bases are not shuffle. It made the first half one base and the other half other base.Ex: The first 100 is one base and from 101 to 200 is the other base. Is there a way to shuffle this?

**Comment by bolshoytoster** [comment]
@elpigeontest yes, in `src/config.js` you can set `shuffleLayerConfigurations` to `true`.

**Comment by elpigeontest** [comment]
> @elpigeontest yes, in `src/config.js` you can set `shuffleLayerConfigurations` to `true`.

Thanks! Works fine!
